pages/api/members.ts

```diff
@@ -39,14 +65,35 @@ export default async function handler(req, res) {
     .map((member: NotionMember) => ({
       name: member?.properties?.Name?.title?.at(0)?.plain_text,
       image: member?.icon?.file?.url,
-      roles: member?.properties?.Role.multi_select.map((role) => role.name),
+      roles: member?.properties?.Role.multi_select
+        .map((role) => role.name)
+        .sort((a, b) => {
+          if (a == "President" && b !== "President") {
+            return -1;
+          } else if (a == "Teamlead" && b !== "Teamlead") {
+            return -1;
+          } else {
+            return 1;
+          }
+        }),
       departments: member?.properties?.[
         "Functional or Mission-Based Department"
       ].multi_select.map((department) => department.name),
       description: "",
       degree: member?.properties?.Degree?.select?.name,
     }))
-    .sort((a, b) => (a.roles[0] < b.roles[0] ? 1 : -1))
+    .sort((a, b) => {
+      if (a.roles.includes("President") && !b.roles.includes("President")) {
```

Comment:
here you could use the min function to get the most important role.

const comparator = (a,b) => Math.min(a.roles.map(role => roleOrder(role))) - Math.min(b.roles.map(role => roleOrder(role)))